### PR TITLE
[Home] Add More Content

### PR DESCRIPTION
**Pull request by imgntn**
This PR adds more content to the "Home" space:

- Fire pit that you can toss objects into
- Various moustaches and beards in the dressing room
- A bench in the hallway
- An umbrella outside
- Removes blue chair
- Swaps switches in living room
- Removes generic block pile from table
- Adds blocky game and 'advance' swipe button
- New artemis model with jacket
- Adds milk pail game and balls to reset script
- Adds target with auto reset after collision and tipped over to ping pong gun

To test:
0. Stop all scripts and load the defaultScripts.js file that comes with this install
1. Visit forum
2. Reload all scripts (Running scripts / Ctrl-R if advanced settings is enabled)
3. Click the tidy guy
4. Observe that objects above reset as expected

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: d50681243c1c14f09b03ce1a7b650c525a7eeae3 available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: bc29bcf0d2ce6fc706d952b58f5c41c13669f0fc available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 69d38214241c85828b8ba51e1dcb072529ca1cd9 available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: fdb0fa45597aeb9547cb39c865432510e10156d6 available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: f2f911ef29017f4b75c9606361da0fda2e5700e9 available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7f89ded241bd3692007c1289a1f4c8f3074c5399 available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6bc745ee017517bc625e22685026fb7f2cef3f63 available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5211b6ad77084527724e37ab19247ee495bb2aed available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 266dfcd8e941cc68e134274c7e5cb65b047d021f available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7cf56401f5ccd1a80792b0393943bdacd87d6624 available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: c7ae3396ebae20e1216afee4bdbcd90d975f9bfe available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: f1e826b3050bc22e28be39f1e1f18a4bfcd49c38 available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3bc2725844891fab6ae14b77059c88d1220d734a available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: b171b5e9eb75e1c0c85b69e90ef083bad84d26ce available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: f6644ad2bb4692f75bcf8f78ed41c2b8aaf1e2c3 available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9c63a6417e709dde9b2fa2fae80630802f34887f available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e87cb2bd541f35661f30ca0838b4c7fe4f9f5a85 available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 27594e6df2bdc0bc8a0ce943cb52b823296f5ac9 available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 47a84fe91d4e92233dfce7f3e9a86e10eae5aa5a available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: eb3cafb4f93f2ef64c584cfc18ed71119f91ed02 available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3603c8cf2ec2d0067dc22112203563e88d7ae983 available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 15745eb5b24aa260aef117015c591029c1b07a81 available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 577142a4d1a3e4d693d9499abe493c5797922b5b available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by kevinmthomas-carpool** [comment]
testing

**Comment by kevinmthomas-carpool** [comment]
## QA Test
**QA Tester**: KevinMThomas
**QA Pass**: **QA Pass** SUCCESS
**Additional Test Results + Observations**: Successful test. Visited ```music``` and  stopped all scripts and loaded the ```defaultScripts.js``` file that came with the PR.  Visited ```forum``` and reloaded all scripts.  Verified new objects mentioned in PR and successfully manipulated them as they behaved properly with normal physical behaviors.  Clicked the tidy guy and observed that objects were reset as expected.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9f4289ae1b164391e4a44b81e3d790fe2088bed9 available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6573b3c71e72da122c337a9ff6ce2353cfb8bac7 available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: c545f53c75041280d40bd7a1f1e7007c2c7e1a29 available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

File: scripts/system/attachedEntitiesManager.js [hunk 1]
```diff
@@ -246,27 +247,27 @@ function AttachedEntitiesManager() {
...
-                if ("grabKey" in parsedUserData) {
-                    parsedUserData.grabKey.refCount = 0;
-                    delete parsedUserData.grabKey["avatarId"];
-                    props["userData"] = JSON.stringify(parsedUserData);
-                }
-            } catch (e) {
-            }
-        }
-    }
+    // this.scrubProperties = function(props) {
```
Comment (ZappoMan) [inline scripts/system/attachedEntitiesManager.js]: did you mean to make these changes?

File: scripts/system/attachedEntitiesManager.js [hunk 2]
```diff
@@ -220,16 +221,16 @@ function AttachedEntitiesManager() {
...
-        print("toggleLocked");
-        if (clothingLocked) {
-            clothingLocked = false;
-            toolBar.setImageURL(Script.resolvePath("assets/images/unlock.svg"), lockButton);
-        } else {
-            clothingLocked = true;
-            toolBar.setImageURL(Script.resolvePath("assets/images/lock.svg"), lockButton);
-        }
-    }
+    // this.toggleLocked = function() {
```
Comment (ZappoMan) [inline scripts/system/attachedEntitiesManager.js]: did you mean to make these changes?

File: scripts/system/attachedEntitiesManager.js [hunk 3]
```diff
@@ -71,12 +71,11 @@ Script.scriptEnding.connect(scriptEnding);
 
 
 
-
 // attached entites
 
 
 function AttachedEntitiesManager() {
-    var clothingLocked = true;
+    var clothingLocked = false;
```
Comment (ZappoMan) [inline scripts/system/attachedEntitiesManager.js]: did you mean to make these changes?

File: scripts/defaultScripts.js [hunk 4]
```diff
@@ -21,3 +21,4 @@ Script.load("system/controllers/handControllerPointer.js");
 Script.load("system/controllers/squeezeHands.js");
 Script.load("system/controllers/grab.js");
 Script.load("system/dialTone.js");
+// Script.load("system/attachedEntitiesManager.js");
```
Comment (ZappoMan) [inline scripts/defaultScripts.js]: did you mean to make these changes?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 65e2cdbbf366a75b5cc70f36014e314392585947 available at: https://gist.github.com/4f12544b413baba61ad298e12362140a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7924.dmg

**Pull request merged**